changelog/fix-6667-disputes-listing-add-respond-button

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: add
+
+Add respond button on dispute list view. Part of a larger set of changes for disputes.
```

Comment:
Since changelog will be added in the [epic](https://github.com/Automattic/woocommerce-payments/issues/6355) level, this changelog should be just a comment.

Reply:
Fixed in [01d2eb8](https://github.com/Automattic/woocommerce-payments/pull/6695/commits/01d2eb8140099bdaeca13eab4c8dd48226e90a95).